src/re_riak.erl

```diff
@@ -758,12 +758,24 @@ nodes(Cluster) ->
     end.
 
 node_config(_, Node) ->
-    load_patch(Node),
-    case remote(Node, re_riak_patch, effective_config, []) of
-        {error, legacy_config} ->
-            [{error, not_found, [{error, <<"Legacy configuration files found, effective config not available.">>}]}];
-        Config ->
-            [{config, Config}]
+    case load_patch(Node) of
```

Comment:
@drewkerrigan I'm thinking that the bare `load_patch(Node)` calls are not a good idea because it can fail if the node passed in isn't good.  Probably should fix the other instance of this in `bucket_types/1`.